src/Stack/New.hs

```diff
@@ -157,24 +157,26 @@ loadTemplate name logIt = do
         downloadTemplate req (templateDir </> rel)
     downloadTemplate :: Request -> Path Abs File -> RIO env Text
     downloadTemplate req path = do
-        fs <- liftIO $ getFileStatus (toFilePath path)
-        let (COff size) = fileSize fs
-            downloadFileSize = fromIntegral size
-            dReq = DownloadRequest
-              { drRequest = req
-              , drHashChecks = []
-              , drLengthCheck = Just downloadFileSize
-              , drRetryPolicy = drRetryPolicyDefault
-              }
+        let path' = toFilePath path
+        exist <- liftIO $ fileExist path'
+
+        downloadFileSize <- if exist
```

Comment:
Maybe I misunderstood our discussions before, but why do we care about the size of the file on the filesystem?

Reply:
The file size is also being use here https://github.com/commercialhaskell/stack/blob/cba6996791c2a7253388acab1b056aa5d30fe35e/src/Network/HTTP/StackClient.hs#L177 as the total size for the progress, I was thinking  to not change the behavior

Reply:
But isn't that known to be wrong?

Reply:
Yeah that's true, OK I'll remove it